### PR TITLE
Update 2021 links inside 2022 levels

### PR DESCRIPTION
**Pull request by davidsbailey**
Continues [PLAT-1861](https://codedotorg.atlassian.net/browse/PLAT-1861). Depends on https://github.com/code-dot-org/code-dot-org/pull/47014. This PR looks specifically at links within levels that have `2022` in the name, because those seem safe to update before 2022 courses are marked as stable. Updates in this PR fall into 2 categories:
1.update links to 2021 lessons/levels which have moved in the 2022 curriculum (https://github.com/code-dot-org/code-dot-org/pull/47003/commits/581282056fc6a2283d18544d276366650c147d2d)
2. fix remaining links from 2022 levels to 2021 lessons/levels which just require replacing `2021` with `2022` (https://github.com/code-dot-org/code-dot-org/pull/47003/commits/938ca768a6128be8eae3092ac25b62f17a9fb18c)

here's what I did:
* made a [spreadsheet](https://docs.google.com/spreadsheets/d/139bb4ID2IBFf-6491I_JfM3CIP36WA6BAy_YQc1uing/edit#gid=0) of all levels matching these search criteria: 
Screen Shot 2022-06-27 at 3 54 11 PM
* removed rows for k5-onlinepd-2021, because there is no k5-onlinepd-2022
* checked whether replacing 2021 with 2022 resulted in a same / similar level or lesson, and noted it in the spreadsheet.
  * those which had moved in 2022 were fixed in step 1
  * those which are still in the same place in 2022 were fixed in step 2
  * those which require curriculum writer attention have been deferred as future work 

## Testing story

* manually verified that the only remaining instances of `2021/lessons` in level files with `2022` in the name are either links to k5-onlinepd-2021, or are the 4 exceptions noted in the spreadsheet
* manually verified that all before and after links updated in this PR look the same, with "before" having 2021 in the url and "after" having 2022 in the URL, as shown in the spreadsheet

## Deployment strategy

safe to merge today

## Follow-up work

* with help from curriculum / PL, address a few problem levels / lessons noted in the spreadsheet
* find and update remaining links to 2021 curriculum, many of which we won't want to update until 6/29
* final check of links in marketing pages (see notes [here](https://docs.google.com/document/d/1RlIniJPlIWXeh4YYDhphlGnYOztMq86En20oaQPhOno/edit#))

### CODE REVIEW

File: dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
     "allow_user_uploads": "false",
```
Comment (davidsbailey) [inline dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level:9]: http --> https

**Comment by dmcavoy** [comment]
There is a k5-onlinepd-2022 https://levelbuilder-studio.code.org/s/k5-onlinepd-2022.

File: dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
```
Comment (dmcavoy) [inline dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level:9]: Not sure what changed here. Can you help me?

Comment (davidsbailey) [reply dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level:9]: Screen Shot 2022-06-28 at 6 46 26 PM

Comment (davidsbailey) [reply dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level:9]: csd5-2021 --> csd5-2022

**Review by dmcavoy — APPROVED** [review]
Looks good as long as we follow up on k5-onlinepd-2022

**Comment by davidsbailey** [comment]
> There is a k5-onlinepd-2022 https://levelbuilder-studio.code.org/s/k5-onlinepd-2022.

oh, whoa! ok, I will tackle those in the next PR.

**Pull request merged**